### DISCUSSION TITLE
pyqt5 vs pyqt6

### DISCUSSION THREAD

**Discussion by ikatkov** [NanoVNA-Saver/nanovna-saver#627] [General]
@zarath 
The source code both reference to pyqt5 and pyqt6. The official version is pyqt6, right?
Assuming this is correct, there are at least 3 places in the code that use pyqt5 syntax.
Is it something that is broken, and should I send a PR, or is it just my wrong understanding?

**Comment by zarath** [comment]
It's in transition from PyQt5 to PyQt6. I know at least one place where further changes have to be done. At least for some stylesheet actions.
Help as PR is always apreciated.
